src/main/java/BasicJavaHomeWork/Exercise_4.java

```diff
@@ -0,0 +1,42 @@
+package BasicJavaHomeWork;
+
+import java.util.Scanner;
+
+public class Exercise_4 {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Please enter a number: ");
+        int numb = in.nextInt();
+        int a;
+        int b;
+        int c = 0;
+
+        for (a=1; a<=numb;a++) {
```

Comment:
you could declare the variable a inside the for loop because you won't need it elsewhere anyway
for(int a = 1; a <= numb; a++){}

Reply:
Thank you for your review, I've changed the declaration of the variable a.